javascript_client/src/subscriptions/createAblyHandler.ts

```diff
@@ -123,6 +121,8 @@ function createAblyHandler(options: AblyHandlerOptions) {
         }
         // When you get an update from ably, give it to Relay
         channel.subscribe("update", updateHandler)
+
+        dispatchResult(response.body)
```

Comment:
```suggestion
        // If this body contains errors, Relay will immediately clean up the subscription. 
        // To simplify the channel state, only dispatch the result _after_ the channel is set up. 
        // That way, if Relay tries to clean it up, it will get cleaned up properly.
        dispatchResult(response.body)
```

Reply:
I'll add this before publishing